src/PurgeClient/LocalPurgeClient.php

```diff
@@ -55,6 +55,8 @@ function ($tag) {
      */
     public function purgeAll()
     {
-        $this->cacheStore->purgeAllContent();
+        $purgeRequest = Request::create('http://localhost/', 'PURGE');
```

Comment:
Hmm, doesn't actual host come from some parameter? At first glance I thought it was a test.

Reply:
This is same as in purge() above, which comes from https://github.com/ezsystems/ezpublish-kernel/blob/1bdad65810d562c8a07be128f1eeb79ec1a5cfa3/eZ/Publish/Core/MVC/Symfony/Cache/Http/LocalPurgeClient.php#L47

reason is host does not matter with the symfony proxy implementation, it's local in the same code base.